### PR TITLE
New product popular adaptive

### PR DESCRIPTION
**Pull request by bynikitachel**

### CODE REVIEW

File: src/components/Gallery/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import Card from '../Card'
+import UsedButton from '../UsedButton'
+
+const Gallery = ({ items, headerText }) => {
+  return (
+    <section className="gallery">
+      <div className="gallery-container">
+        <div className="gallery-container__header">
+          <HeaderSection headerText={headerText} />
+          <UsedButton
```
Comment (imma-fish-u) [inline src/components/Gallery/index.js]: Try to use react-responsive library with mediaquery hook (I used this: const isTabletOrMobile = useMediaQuery({ query: "(max-width: 1200px)" }); ) and simple button component with if else statement. (isTabletOrMobile) ? Button primary.... : Button secondary

File: src/components/Gallery/index.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.gallery {
+    .gallery-container {
```
Comment (imma-fish-u) [inline src/components/Gallery/index.scss:2]: gallery__container would be better

File: src/components/Gallery/index.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        &__content {
+            display: flex;
+            overflow-x: auto;
+            margin: 36px 0 132px;
+        }
+    }
+}
+
+@media (max-width: 768px) {
```
Comment (imma-fish-u) [inline src/components/Gallery/index.scss]: we put media query inside of every selector and pls use variables I created for media query to stay in tune

File: src/components/UsedButton/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                      firstArrowRight,
+                      FirstArrowLeft,
+                      secondArrowRight,
+                      secondArrowLeft,
+                    }) => {
+
+  const [ isDesktop, setDesktop ] = useState(window.innerWidth > 767)
+
+  const updateMedia = () => {
+    setDesktop(window.innerWidth > 767)
```
Comment (imma-fish-u) [inline src/components/UsedButton/index.js]: so react responsive lib will ease your life and you don't have to write such code anymore

File: src/components/WhyWeAreItem/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const WhyWeAreItem = ({ image, text }) => {
+  return (
+    <div className="why-we-are-item">
+      <div className="why-we-are-item__image">
+        <img
+          src={require(`../../assets/icons/WhyWeAreIcons/${image}`)}
+          width="66pх"
```
Comment (imma-fish-u) [inline src/components/WhyWeAreItem/index.js]: pls give a classname to this img and move width and height in scss

File: src/pages/home/sections/Popular/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Gallery from '../../../../components/Gallery'
+
+const Index = ({ popularData }) => {
```
Comment (imma-fish-u) [inline src/pages/home/sections/Popular/index.js]: Lets try nicer naming for component, index doesn't really say what is it about

**Review by imma-fish-u — CHANGES_REQUESTED** [review]

File: src/components/Gallery/index.scss (hunk 2 above)
Comment (bynikitachel) [reply src/components/Gallery/index.scss:2]: why?

**Pull request closed without merging**